packages/react-core/src/Authenticator/hooks/useAuthenticatorInitMachine/useAuthenticatorInitMachine.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { AuthenticatorMachineOptions } from '@aws-amplify/ui';
+
+import { useAuthenticator } from '../useAuthenticator';
+
+// Utility hook that sends init event to the parent provider
```

Comment:
Now that this no longer lives with the `Authenticator` UI (component and the provider) this comment isn't as relevant. What do you think of removing or expanding it?